addons/danger/XEH_PREP.hpp

```diff
@@ -1,25 +1,13 @@
 PREP(assault);
 PREP(assaultBuilding);
 PREP(assaultSpeed);
-PREP(checkBody);
-PREP(coward);
-PREP(doCallout);
-PREP(doUGL);
-PREP(doSmoke);
-PREP(findBuildings);
-PREP(findCover);
-PREP(findOverwatch);
-PREP(findReadyUnits);
+PREP(doCheckBody);
+PREP(isCoward);
 PREP(fleeing);
 PREP(fsmAllowAnimation);
 PREP(fsmExitVariables);
-PREP(eventCallback);
-PREP(gesture);
-PREP(hideInside);
+PREP(doHide);
 PREP(immediateAction);
```

Comment:
This should probably be "doImmediateAction", though in truth, it would be better named "doDodge"